build-pipeline.yml

```diff
@@ -50,3 +50,22 @@ jobs:
     parameters:
       # pass through all parameters
       ${{ insert }}: ${{ parameters }}
+
+# If the build was launched because a tag was created, we'll want to deploy
+# this to the AppStore.
+- ${{ if startsWith( variables['Build.SourceBranch'], 'refs/tags') }}:
+  # Repositories using our build pipeline outside of the
+  # shotgunsoftware organization can be tagged, but they can't
+  # use our deploy scripts as they do not have the necessary
+  # permissions to access our secrets, as they should.
+  #
+  # Therefore, we need to avoid including this template for repos that are
+  # not under the shotgunsoftware organization.
+  #
+  # FIXME: There is no way to tell if an Azure build is running under
+  # our organization or not. What we can do however is check the name
+  # of the Azure project from which this build is launched.
```

Comment:
I'm not clear on the scope here. Is this saying that the build process can be triggered by 3rd party repos, that contains our azure config?
This may be fine, but I'm not sure I understand enough around the process here, happy to chat about it if you want.